packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -218,6 +233,16 @@ function isMatchingRootContainer(
   );
 }
 
+export function isDOMElement(target: EventTarget): boolean {
+  const nodeType = ((target: any): Node).nodeType;
+  return nodeType != null && nodeType === ELEMENT_NODE;
```

Comment:
Isn't the `!= null` check here unnecessary?

Reply:
Flow complains otherwise.

Reply:
Ugh. Cast it.